Jenkinsfile_CNP

```diff
@@ -69,6 +65,18 @@ withPipeline(type, product, component) {
   }
 
   onMaster {
+    secrets = [
+      (appVault): [
```

Comment:
The secrets are already defined globally at the top, please remove this from here